kedro/ipython/__init__.py

```diff
@@ -61,7 +62,7 @@ def load_ipython_extension(ipython):
 @argument("-e", "--env", type=str, default=None, help=ENV_HELP)
 @argument(
     "--params",
-    type=lambda value: _split_params(None, None, value),
+    type=str,
```

Comment:
would `_split_params()` not work here still?

Reply:
I don't think so since `_split_params` needs the click context.

Reply:
It doesn't really use `ctx`, and `param` anywhere, so they can be set to `None` as they were originally. I meant that the params will be processed by `_split_params` into a `dict` already. So this file can remain unchanged!

Reply:
Right I see what you are saying.